doc/installation/pip.rst

```diff
@@ -5,67 +5,125 @@ Python Package Index
 subdirectory.
 
-privacyIDEA can run with Python 2.7 and 3.5, 3.6 or 3.7. Other versions either do not work
-or are not tested.
+privacyIDEA currently runs with Python 2.7 and 3.5, 3.6, 3.7 and 3.8. Other
+versions either do not work or are not tested.
 
-You first need to install some development packages. E.g. on Debian-based
-distributions like Ubuntu the packages are called
+You first need to install a package for creating a python `virtual environment
+<https://virtualenv.pypa.io/en/stable/>`_.
 
-* libjpeg-dev
-* libz-dev
-* python-dev
-* libffi-dev
-* libssl-dev
-* libxslt1-dev
-
-Eventually you may have to install database-related packages such as
-
-* libpq-dev
-
-Now you can install privacyIDEA like this::
+Now you can setup the virtual environment for privacyIDEA like this::
 
   virtualenv /opt/privacyidea
 
   cd /opt/privacyidea
   source bin/activate
 
-Now you are within the python virtual environment.
-Within the environment you can now run::
- 
+.. note::
+    Some distributions still ship Python 2.7 as the system python. If you want
+    to use Python 3 you can create the virtual environment like this:
+    `virtualenv -p /usr/bin/python3 /opt/privacyidea`
+
+Now you are within the python virtual environment and you can run::
+
   pip install privacyidea
 
-To achieve a deterministic installation, you can now pin the installed
-versions to our tested versions like this::
+in order to install the latest privacyIDEA version from
+`PyPI <https://pypi.org/project/privacyIDEA>`_.
+
+.. note::
+    The `setup.py` of version 3.3.3 is missing the package `cbor2`. privacyIDEA
+    won't work until the `cbor2`-package is installed in the virtual environment.
+
+To achieve a deterministic installation, you can install and pin the installed
+versions like this::
+
+  export PI_VERSION=3.3.3
+  pip install -r https://raw.githubusercontent.com/privacyidea/privacyidea/v${PI_VERSION}/requirements.txt
+  pip install git+https://github.com/privacyidea/privacyidea.git@v${PI_VERSION}
+
+.. note::
+    The requirements file for version 3.3.3 pins a version of the
+    `cbor2` package which requires to rebuild some components. In order to
+    install the pinned version, the system packages `gcc` and `python-dev` must
+    be installed.
+
+Configuration
+.............
+
+Database
+^^^^^^^^
+
+privacyIDEA makes use of `SQLAlchemy <https://www.sqlalchemy.org>`_ to be able
+to talk to different SQL-based databases. Our best experience is with
+`MySQL <https://www.mysql.com/>`_ but SQLAlchemy supports many different
+databases [#sqlaDialects]_.
+
+The database server should be installed on the host or be otherwise reachable.
+
+In order to use the database, a user with the appropriate privileges is needed::
+
+    CREATE DATABASE pi;
+    CREATE USER "pi"@"localhost" IDENTIFIED BY "<dbsecret>";
+    GRANT ALL PRIVILEGES ON pi.* TO "pi"@"localhost";
+
+The database name, user and password are then added to :ref:`cfgfile`.
```

Comment:
I think this is an important step here, which needs to be more specifc.

This sentence could sound like "these values are automatically added".
A sense like

"You need to add the database settings to the pi.cfg file. For more information on this config file read..."

The text currently does not address the reader (you) but uses a passive form. But it must be clear, that the guy who is currently setting up the system needs to work on the pi.cfg file here.

Reply:
done in e814415f

Reply:
nice. thx.